ReactNative/ReactTwitter/core/views/main-navigator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {
+  Navigator
+} from 'react-native'
+
+var DebugScreenView = require('./debug-screen.js')
+var DeepLinkingView = require('./deep-linking.js')
+
+const debugScreenID = 'debug-screen-identifier'
```

Comment:
You could put them in a "constant" module and export them, then get them with `require('../MyConstants').DEBUG_SCREEN_ID`.